interface/forms/group_attendance/report.php

```diff
@@ -38,6 +41,35 @@ function group_attendance_report($pid, $encounter, $cols, $id)
                 <td><span class='text'><?php echo text($group_name); ?></span></td>
             </tr>
         </table>
+        <table class="table table-bordered w-100">
+            <tr>
+                <td><span class='font-weight-bold'><?php echo xlt('Participant'); ?></span></td>
+                <td><span class='font-weight-bold'><?php echo xlt('Status'); ?></span></td>
+                <td><span class='font-weight-bold'><?php echo xlt('Comments'); ?></span></td>
+            </tr>
+            <?php
+            if ($participants) {
+                foreach ($participants as $participant) {
+                    $name = $participant['lname'] . ', ' . $participant['fname'];
+                    $attnStatus = getAttendanceStatus($participant['meeting_patient_status'])
+                    ?>
+                    <tr>
+                        <td><span class='text'><?php echo text($name); ?></span></td>
+                        <td><span class='text'><?php echo text($attnStatus); ?></span></td>
```

Comment:
This is from a list, so should translate with the xl_list_label() function:
```
<td><span class='text'><?php echo text(xl_list_label($attnStatus)); ?></span></td>
```

Reply:
Bump..